<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="10" ignored="3" total="23" passed="10">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="25998" started-at="2021-06-18T12:14:19Z" finished-at="2021-06-18T12:14:45Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="25998" started-at="2021-06-18T12:14:19Z" finished-at="2021-06-18T12:14:45Z">
      <class name="StripeApitestingScript.postRequest">
        <test-method status="PASS" signature="postRequest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:StripeApitestingScript.postRequest@6eda5c9]" name="postRequest" duration-ms="734" started-at="2021-06-18T12:14:42Z" data-provider="data" finished-at="2021-06-18T12:14:43Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[cus_Jgr1XgpyulZJlb]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[qaaa@gmail.com]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[ggdgdgdgd]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- postRequest -->
        <test-method status="PASS" signature="postRequest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:StripeApitestingScript.postRequest@6eda5c9]" name="postRequest" duration-ms="595" started-at="2021-06-18T12:14:43Z" data-provider="data" finished-at="2021-06-18T12:14:43Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[cus_JgqswSIFdCz3uZ]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[qaaa@gmail.com]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[ggdgdgdgd]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- postRequest -->
        <test-method status="PASS" signature="postRequest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:StripeApitestingScript.postRequest@6eda5c9]" name="postRequest" duration-ms="535" started-at="2021-06-18T12:14:43Z" data-provider="data" finished-at="2021-06-18T12:14:44Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[cus_JgqJ25UCJQ0aTy]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[qaaa@gmail.com]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[ggdgdgdgd]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- postRequest -->
        <test-method status="PASS" signature="postRequest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:StripeApitestingScript.postRequest@6eda5c9]" name="postRequest" duration-ms="594" started-at="2021-06-18T12:14:44Z" data-provider="data" finished-at="2021-06-18T12:14:44Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[cus_JgqIAizgRs4D1a]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[qaaa@gmail.com]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[ggdgdgdgd]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- postRequest -->
        <test-method status="PASS" signature="postRequest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:StripeApitestingScript.postRequest@6eda5c9]" name="postRequest" duration-ms="568" started-at="2021-06-18T12:14:44Z" data-provider="data" finished-at="2021-06-18T12:14:45Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[cus_JgppqmwHCIgW3q]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[qaaa@gmail.com]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[ggdgdgdgd]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- postRequest -->
      </class> <!-- StripeApitestingScript.postRequest -->
      <class name="StripeApitestingScript.getRequest">
        <test-method status="PASS" signature="basic()[pri:0, instance:StripeApitestingScript.getRequest@20ce78ec]" name="basic" is-config="true" duration-ms="1242" started-at="2021-06-18T12:14:19Z" finished-at="2021-06-18T12:14:20Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- basic -->
        <test-method status="PASS" signature="getRequest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:StripeApitestingScript.getRequest@20ce78ec]" name="getRequest" duration-ms="3896" started-at="2021-06-18T12:14:24Z" data-provider="data" finished-at="2021-06-18T12:14:28Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[cus_Jh46hl7jE4ajLv]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[qaaa@gmail.com]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[ggdgdgdgd]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getRequest -->
        <test-method status="PASS" signature="getRequest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:StripeApitestingScript.getRequest@20ce78ec]" name="getRequest" duration-ms="557" started-at="2021-06-18T12:14:28Z" data-provider="data" finished-at="2021-06-18T12:14:28Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[cus_Jh46uY2luJ0kfs]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[qaaa@gmail.com]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[ggdgdgdgd]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getRequest -->
        <test-method status="PASS" signature="getRequest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:StripeApitestingScript.getRequest@20ce78ec]" name="getRequest" duration-ms="548" started-at="2021-06-18T12:14:28Z" data-provider="data" finished-at="2021-06-18T12:14:29Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[cus_Jh464GXnn8mHMo]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[qaaa@gmail.com]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[ggdgdgdgd]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getRequest -->
        <test-method status="PASS" signature="getRequest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:StripeApitestingScript.getRequest@20ce78ec]" name="getRequest" duration-ms="546" started-at="2021-06-18T12:14:29Z" data-provider="data" finished-at="2021-06-18T12:14:30Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[cus_Jh469YFnZyNdTm]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[qaaa@gmail.com]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[ggdgdgdgd]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getRequest -->
        <test-method status="PASS" signature="getRequest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:StripeApitestingScript.getRequest@20ce78ec]" name="getRequest" duration-ms="558" started-at="2021-06-18T12:14:30Z" data-provider="data" finished-at="2021-06-18T12:14:30Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[cus_Jh4612Ujsb7frx]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[qaaa@gmail.com]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[ggdgdgdgd]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getRequest -->
      </class> <!-- StripeApitestingScript.getRequest -->
      <class name="StripeApitestingScript.deleteRequest">
        <test-method status="FAIL" signature="deleteRequest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:StripeApitestingScript.deleteRequest@56620197]" name="deleteRequest" duration-ms="702" started-at="2021-06-18T12:14:37Z" data-provider="data" finished-at="2021-06-18T12:14:38Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[cus_Jh46hl7jE4ajLv]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[qaaa@gmail.com]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[ggdgdgdgd]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Request filed mismatch of status expected [200] but found [404]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Request filed mismatch of status expected [200] but found [404]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at StripeApitestingScript.deleteRequest.deleteRequest(deleteRequest.java:25)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deleteRequest -->
        <test-method status="FAIL" signature="deleteRequest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:StripeApitestingScript.deleteRequest@56620197]" name="deleteRequest" duration-ms="717" started-at="2021-06-18T12:14:38Z" data-provider="data" finished-at="2021-06-18T12:14:38Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[cus_Jh46uY2luJ0kfs]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[qaaa@gmail.com]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[ggdgdgdgd]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Request filed mismatch of status expected [200] but found [404]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Request filed mismatch of status expected [200] but found [404]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at StripeApitestingScript.deleteRequest.deleteRequest(deleteRequest.java:25)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deleteRequest -->
        <test-method status="FAIL" signature="deleteRequest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:StripeApitestingScript.deleteRequest@56620197]" name="deleteRequest" duration-ms="699" started-at="2021-06-18T12:14:38Z" data-provider="data" finished-at="2021-06-18T12:14:39Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[cus_Jh464GXnn8mHMo]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[qaaa@gmail.com]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[ggdgdgdgd]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Request filed mismatch of status expected [200] but found [404]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Request filed mismatch of status expected [200] but found [404]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at StripeApitestingScript.deleteRequest.deleteRequest(deleteRequest.java:25)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deleteRequest -->
        <test-method status="FAIL" signature="deleteRequest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:StripeApitestingScript.deleteRequest@56620197]" name="deleteRequest" duration-ms="691" started-at="2021-06-18T12:14:39Z" data-provider="data" finished-at="2021-06-18T12:14:40Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[cus_Jh469YFnZyNdTm]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[qaaa@gmail.com]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[ggdgdgdgd]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Request filed mismatch of status expected [200] but found [404]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Request filed mismatch of status expected [200] but found [404]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at StripeApitestingScript.deleteRequest.deleteRequest(deleteRequest.java:25)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deleteRequest -->
        <test-method status="FAIL" signature="deleteRequest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:StripeApitestingScript.deleteRequest@56620197]" name="deleteRequest" duration-ms="735" started-at="2021-06-18T12:14:40Z" data-provider="data" finished-at="2021-06-18T12:14:41Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[cus_Jh4612Ujsb7frx]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[qaaa@gmail.com]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[ggdgdgdgd]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Request filed mismatch of status expected [200] but found [404]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Request filed mismatch of status expected [200] but found [404]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at StripeApitestingScript.deleteRequest.deleteRequest(deleteRequest.java:25)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deleteRequest -->
      </class> <!-- StripeApitestingScript.deleteRequest -->
      <class name="StripeApitestingScript.putRequest">
        <test-method status="FAIL" signature="putRequest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:StripeApitestingScript.putRequest@393671df]" name="putRequest" duration-ms="692" started-at="2021-06-18T12:14:32Z" data-provider="data" finished-at="2021-06-18T12:14:33Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[cus_Jh46hl7jE4ajLv]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[qaaa@gmail.com]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[ggdgdgdgd]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Request filed mismatch of status expected [200] but found [400]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Request filed mismatch of status expected [200] but found [400]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at StripeApitestingScript.putRequest.putRequest(putRequest.java:26)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- putRequest -->
        <test-method status="FAIL" signature="putRequest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:StripeApitestingScript.putRequest@393671df]" name="putRequest" duration-ms="488" started-at="2021-06-18T12:14:33Z" data-provider="data" finished-at="2021-06-18T12:14:33Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[cus_Jh46uY2luJ0kfs]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[qaaa@gmail.com]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[ggdgdgdgd]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Request filed mismatch of status expected [200] but found [400]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Request filed mismatch of status expected [200] but found [400]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at StripeApitestingScript.putRequest.putRequest(putRequest.java:26)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- putRequest -->
        <test-method status="FAIL" signature="putRequest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:StripeApitestingScript.putRequest@393671df]" name="putRequest" duration-ms="510" started-at="2021-06-18T12:14:34Z" data-provider="data" finished-at="2021-06-18T12:14:34Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[cus_Jh464GXnn8mHMo]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[qaaa@gmail.com]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[ggdgdgdgd]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Request filed mismatch of status expected [200] but found [400]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Request filed mismatch of status expected [200] but found [400]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at StripeApitestingScript.putRequest.putRequest(putRequest.java:26)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- putRequest -->
        <test-method status="FAIL" signature="putRequest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:StripeApitestingScript.putRequest@393671df]" name="putRequest" duration-ms="516" started-at="2021-06-18T12:14:34Z" data-provider="data" finished-at="2021-06-18T12:14:35Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[cus_Jh469YFnZyNdTm]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[qaaa@gmail.com]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[ggdgdgdgd]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Request filed mismatch of status expected [200] but found [400]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Request filed mismatch of status expected [200] but found [400]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at StripeApitestingScript.putRequest.putRequest(putRequest.java:26)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- putRequest -->
        <test-method status="FAIL" signature="putRequest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:StripeApitestingScript.putRequest@393671df]" name="putRequest" duration-ms="516" started-at="2021-06-18T12:14:35Z" data-provider="data" finished-at="2021-06-18T12:14:35Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[cus_Jh4612Ujsb7frx]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[qaaa@gmail.com]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[ggdgdgdgd]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Request filed mismatch of status expected [200] but found [400]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Request filed mismatch of status expected [200] but found [400]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at StripeApitestingScript.putRequest.putRequest(putRequest.java:26)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- putRequest -->
      </class> <!-- StripeApitestingScript.putRequest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
